src/package_Swing/mongoContacts.py

```diff
@@ -106,7 +107,7 @@ def setSex(self, sex) -> None:
 def add_Data(new_phone, new_sex, new_name) -> None:
     
     # check if the contact item already exists 
-    if contactData.count_documents({"Phone": new_phone}) > 0:
```

Comment:
except ValueError:
                print("Invalid format")

        elif upper == 'M' or upper == 'MALE':
            self.sex = 'Male'
        else: print("Invalid format")


Line 93/102:  Such "print()" code should not be in data class.  Was the seetPhone()/setSex() call successful if those lines were executed?